tests/Benchmarks/BufferWriterBench.cs

```diff
@@ -0,0 +1,527 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Xunit.Performance;
+using System;
+using System.Buffers;
+using System.Buffers.Text;
+using System.Buffers.Writer;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.Utf8;
+using System.Threading;
+
+public class BufferWriterBench
+{
+    private static AsciiString _crlf = "\r\n";
```

Comment:
nit: static fields should be prefixed with `s_`

Reply:
wil do